microsoftgraph/msgraph-cli-archived
# Task/generate group command

## Overview

1. Adds a directive to remove invalid operation `/groups/microsoft.graph.validateProperties` from the OpenAPI document for the group extension. The correct operation is `/groups/{group-id}/microsoft.graph.validateProperties`
2. Adds the generated groups extension to the list of command modules 

### Demo

N/A

### Notes

N/A
## Testing Instructions

* Pull these changes
* Run the `generate_extensions.py` script
* Observe that the groups command generates successfully.

Fixes #138

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoftgraph/msgraph-cli/pull/180)
